docs/user_guide/disconnected/index.md

```diff
@@ -4,286 +4,52 @@ title: Disconnected
+If you are working in an air-gapped network environment where some of your
+Katello servers do not have Internet connectivity, you may be interested in
+using the Katello Inter-Server Sync feature. This allows you to export repos
+(including repos in content views) on your "upstream" Katello server, and then
+import said repos into your "downstream" server that does not have
+connectivity.
 
-![Disconnected Workflow](./katello-disconnected.png)
+It is possible to export either individual yum repos, or all yum repos in a
+particular content view. For now, only yum content is supported. Future
+releases may include support for more content types such as Puppet modules.
 
-*katello-disconnected* is used via a command line interface with a summary outlined here:
+![ISS Workflow](./ISS.png)
 
+The diagram above shows an example scenario where a user wants to export all
+Yum content in a content view and then import to another Katello server. The
+ISO file is burned to media and then walked across the air-gap in the network.
 
-```
-katello-disconnected(1)        Katello Reference       katello-disconnected(1)
+NOTE: This feature is intended to replace the 'katello-disconnected' script.
 
-NAME
-       katello-disconnected - Tool for fetching content from CDN
+## Detailed Operation
 
-SYNOPSIS
-         katello-disconnected [global_options] [subcommand [options]]
+### Exporting
 
-         Possible commands are:
+The ISS feature allows users to move Yum content from one Katello server to
+another, in a way that is compatible with air-gapped networks. Typically, users
+will set up an upstream server that is connected to the Internet, and then
+create a content view that contains Yum content that they would like to present
+to the downstream server (step 1).
 
-             setup       set pulp configuration (oauth is required)
-             import      import a manifest
-             list        list all or enabled repositories
-             enable      enable particular repository or all repos
-             disable     disable particular repository or all repos
-             configure   create and configure repositories in pulp
-             sync        start synchronizing all or particular repos
-             watch       watch synchronization progress
-             export      export all or particular repos
-             refresh     redownload repository information from CDN
-             clean       remove all repositories from pulp
+At that point, content can be exported via <hammer command>. It will be
+exported to the location set in "pulp_export_destination" in the Settings page,
+under the Katello tab. This defaults to "/tmp/katello-repo-export" (step 2).
```

Comment:
Might be worth informing users here if they want to use a different directory what the selinux/file permissions required on the directory they choose/create need to be.